packages/create-request-form/src/lib/create-request-form.svelte

```diff
@@ -73,6 +73,8 @@
   const submitForm = async (e: Event) => {
     e.preventDefault();
 
+    formData.miscellaneous.builderId = config?.builderId || "request-team";
```

Comment:
"request-team" effectively refers to the Request Finance team.

I think the default should be an empty string and our deployed template should explicitly set "request-network-foundation" as the builderId.

Can we also add `createdWith`?

```suggestion
    formData.miscellaneous.builderId = config?.builderId || "";
    formData.miscellaneous.createWith = window.location.hostname;
```